pkgs/games/ldmud/default.nix

```diff
@@ -0,0 +1,107 @@
+{ lib, fetchFromGitHub, stdenv, autoreconfHook, pkg-config, bison, libiconv
+, pcreSupport ? true, pcre, gcryptSupport ? true, libgcrypt, erqSupport ? "erq"
+, compatModeSupport ? false, filenameSpacesSupport ? false, ipv6Support ? false
+, mccpSupport ? false, zlib, mysqlSupport ? false, libmysqlclient
+, postgresSupport ? false, postgresql, sqliteSupport ? false, sqlite
+, jsonSupport ? false, json_c, xmlSupport ? false, libxml2
+, deprecatedSupport ? true, tlsSupport ? false, openssl, pythonSupport ? false
+, python39, pythonSupportVersion ? python39 }:
+
+let supportedErq = [ "no" "erq" "xerq" ];
+in assert builtins.elem erqSupport supportedErq;
```

Comment:
```suggestion
assert builtins.elem erqSupport [ "no" "erq" "xerq" ];
```

Reply:
This fell away because I removed the option to configure the erq mode and simply defaulted to `xerq`.